data/new

```diff
@@ -0,0 +1,17 @@
+import net from 'net';
```

Comment:
I'm guessing this was some notes/code-along with one of the demos the other day? As far as I can tell this file can be removed.